src/swapd/runtime.rs

```diff
@@ -262,6 +263,7 @@ pub struct CheckpointSwapd {
     pub temporal_safety: TemporalSafety,
     pub txs: HashMap<TxLabel, bitcoin::Transaction>,
     pub txids: HashMap<TxLabel, Txid>,
+    pub pending_broadcasts: Vec<bitcoin::Transaction>,
```

Comment:
I'm not convinced this is needed since that data is in tasks.tasks

Reply:
I prefer it this way, since it is more transparent on what the actual contents of the checkpoint are, as opposed to the more opaque `Task`. Besides, you'll have to re-assemble the task in any case on checkpoint restore, since it will need a new lifetime and id.